components/Tabs/src/private/MDCItemBarCell.m

```diff
@@ -185,12 +185,28 @@ - (void)applyStyle:(MDCItemBarStyle *)style {
     [self updateTitleTextColor];
     [self updateInk];
     [self updateSubviews];
+    [self updateTitleLines];
     [self updateTitleFont];
     [self updateTransformsAnimated:NO];
     [self setNeedsLayout];
   }
 }
 
+- (void)updateTitleLines {
+  // The presence of an image restricts titles to a single line
+  if (_imageView && !_imageView.hidden) {
```

Comment:
Nit: Presence of the image view is more of a secondary factor to decide this. The primary source of truth here should be the tab bar style shouldDisplayImage

Reply:
Done